Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Booking.hs

```diff
@@ -61,20 +63,29 @@ findById bookingId = buildDType $
       where_ $ rb ^. BookingTId ==. val (toKey bookingId)
       pure (rb, bFromLoc, bToLoc, farePars)
 
-findBySearchReq :: Transactionable m => Id DSR.SearchRequest -> m (Maybe Booking)
-findBySearchReq searchReqId = buildDType $
-  fmap (fmap $ extractSolidType @Booking) $
-    Esq.findOne' $ do
-      (rb :& bFromLoc :& bToLoc :& farePars :& dq) <-
-        from
-          ( baseBookingTable
-              `innerJoin` table @DriverQuote.DriverQuoteT
-              `Esq.on` ( \(rb :& _ :& _ :& _ :& dq) ->
-                           rb ^. BookingQuoteId ==. dq ^. DriverQuoteTId
-                       )
-          )
-      where_ $ dq ^. DriverQuoteSearchRequestId ==. val (toKey searchReqId)
-      pure (rb, bFromLoc, bToLoc, farePars)
+findBySearchReq' :: (Transactionable m) => Id DSR.SearchRequest -> m (Maybe Booking)
```

Comment:
why is `'` required in function name? There is only one function of this type, right?